examples/cache-with-aws/README.ja.md

```diff
@@ -0,0 +1,32 @@
+# Java クライアント SDK
```

Comment:
i'm not sure how up-to-date this is. it might be better to remove it and let Erika know that we are missing some translations here and she can decide how to prioritize with the folks who have been helping us translate.

Reply:
I added Erika too for her opinion!

Reply:
Removed the JA readme for now and will sync with her later on this